sqlmesh/dbt/test.py

```diff
@@ -108,14 +108,16 @@ def to_sqlmesh(self, context: DbtContext) -> Audit:
         skip = not self.enabled
         blocking = self.severity == Severity.ERROR
 
-        return Audit(
+        audit = Audit(
             name=self.name,
             dialect=self.dialect,
             skip=skip,
             blocking=blocking,
             query=query,
             jinja_macros=jinja_macros,
         )
+        audit._path = self.path
```

Comment:
@crericha I noticed path wasn't set here so I added it. Please let me know if that will cause problems.